webpack/components/ConfirmModal.js

```diff
@@ -11,6 +11,7 @@ const ConfirmModal = props => {
 
   const actions = [
     <Button
+      ouiaId={`oscap-conf-modal-${props.record.id}-confirm`}
```

Comment:
`props.record` default is null. so this should be  `props.record?.id` or something else

Reply:
Fixed. Although, anything that is fixed here and is not related to the wizard should be removed, since those are leftovers from OVAL additions.